<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>231</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Operating Systems Issues</string>
      </property>
      <widget class="QCheckBox" name="blue_screen">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue screen of death</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="random_shutdowns">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Random shutdowns</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="random_restarts">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Random restarts</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="unknown_processes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Unknown processes</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="device_not_recognized">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Device not recognized</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="settings_reset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Settings reset</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="driver_errors">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Driver errors</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="black_screen_after_driver_update">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Black screen after driver update</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="display_driver_crashes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Display driver crashes</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="blank_screen_with_system_running">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Blank screen with system running</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="disabled_security">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Disabled security</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="pop_ups">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pop ups</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="slow_performance">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Slow performance</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="boot_failure">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Boot failure</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="incorrect_date_time">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Incorrect date time</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="beep_codes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Beep codes</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>211</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Hardware Issues</string>
      </property>
      <widget class="QCheckBox" name="strange_noises">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>94</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Strange noises</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="hot_exterior">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Hot exterior</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="clicking_noises">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Clicking noises</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="audio_problems">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>96</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Audio problems</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="missing_hardware">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>106</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Missing hardware</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="gpu_fan_running_loudly">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Gpu fan running loudly</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="memory_errors">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>93</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Memory errors</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="device_malfunction">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>113</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Device malfunction</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="disk_errors">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>74</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Disk errors</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="corrupted_files">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>94</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Corrupted files</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Monitor Issues</string>
      </property>
      <widget class="QCheckBox" name="display_artifacts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Display artifacts</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="no_display">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>No display</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="flickering_screen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Flickering screen</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="graphical_glitches">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Graphical glitches</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="vertical_lines">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Vertical lines</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="distorted_colors">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Distorted colors</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="dead_pixels">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dead pixels</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>10</y>
     <width>411</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choose The Issues You Have Encountered So Far:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>690</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Reset</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>690</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Network Issues</string>
      </property>
      <widget class="QCheckBox" name="slow_connection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Slow connection</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="network_not_detected">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Network not detected</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="no_internet_access">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>No internet access</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="intermittent_connectivity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Intermittent connectivity</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="browser_redirection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Browser redirection</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Keyboard Issues</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="unresponsive_keys">
         <property name="text">
          <string>Unresponsive keys</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="stuck_keys">
         <property name="text">
          <string>Stuck keys</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="ghost_typing">
         <property name="text">
          <string>Ghost typing</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="delayed_response">
         <property name="text">
          <string>Delayed response</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="keyboard_not_detected">
         <property name="text">
          <string>Keyboard not detected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Performance Issues</string>
      </property>
      <widget class="QCheckBox" name="freezing">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Freezing</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="frequent_crashes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Frequent crashes</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QCheckBox" name="poor_3d_performance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Poor 3D performance</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="application_crashes_with_graphics">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Application crashes with graphics</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="game_crashes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Game crashes</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
